src/main/flight/pid.c

```diff
@@ -79,6 +79,7 @@ typedef struct {
     // Dterm notch filtering
 #ifdef USE_DTERM_NOTCH
     biquadFilter_t deltaNotchFilter;
+    bool isNotchFilterInitialized;
```

Comment:
No, please do not do this. Lazy initialisation is just not appropriate here. What we need to do is ensure the initialisation sequence is correct, not have a work around like this.